libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/ProjectPermissions.java

```diff
@@ -68,7 +68,7 @@ public static boolean isUserInBU(Project document, String bu) {
               && !isNullOrEmpty(document.getBusinessUnit()) && document.getBusinessUnit().startsWith(buFromOrganisation);
     }
 
-    public static boolean userIsEquivalentToModeratorinProject(Project input, String user) {
+    public static boolean userIsEquivalentToModerationProject(Project input, String user) {
```

Comment:
I think this change makes the sitution worse, the better alternative might be `userIsEquivalentToModeratorInProject`